.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+- [ ] Closes #xxxx (Replace xxxx with the GitHub issue number)
```

Comment:
I'll also enforce that the OP provides a PR description, to give a summary of the changes for easier review.

Copying this from one of my other projects:

```
# PR Description

Please describe the changes proposed in the pull request:

-
-
```

Reply:
Good idea, added.